logical_router_lb.go

```diff
@@ -0,0 +1,122 @@
+/**
+ * Copyright (c) 2017 eBay Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+package goovn
+
+import (
+	"fmt"
+	"github.com/ebay/libovsdb"
+)
+
+
+func (odbi *ovnDBImp) lrlbAddImp(lr string, lb string) (*OvnCommand, error) {
+	var operations []libovsdb.Operation
+	namedUUID, err := newRowUUID()
+	if err != nil {
+		return nil, err
+	}
+	mutateUUID := []libovsdb.UUID{{namedUUID}}
+	mutateSet, err := libovsdb.NewOvsSet(mutateUUID)
+	mutation := libovsdb.NewMutation("load_balancer", opInsert, mutateSet)
+	if err != nil {
+		return nil, err
+	}
+	condition := libovsdb.NewCondition("name", "==", lr)
+	mutateOp := libovsdb.Operation{
+		Op:        opMutate,
+		Table:     tableLogicalRouter,
+		Mutations: []interface{}{mutation},
+		Where:     []interface{}{condition},
+	}
+	operations = append(operations, mutateOp)
+	return &OvnCommand{operations, odbi, make([][]map[string]interface{}, len(operations))}, nil
```

Comment:
why you don't check passed data? i think that we need to get from cache data for provided lb and lr and return ErrorNotExist when uuid not in cache. @hzhou8 what you think?

Reply:
I found one issue in UT. populate cache only populates entries for add and delete. cache doesnt update on mutate ops. Need to fix that too. But I am thinking it in separate commit as this itself is big commit. Is it ok ?

Reply:
Agree with @vtolstov validation is needed.

Reply:
yes, for mutate cache update is nice to have in separate pr

Reply:
Ack!